tests/gtest/avifmetadatatest.cc

```diff
@@ -155,40 +155,36 @@ TEST_P(MetadataTest, Read) {
 
 INSTANTIATE_TEST_SUITE_P(
     PngNone, MetadataTest,
-    Combine(Values("paris_exif_xmp_icc.png"),  // zTXt iCCP iTXt IDAT
+    Combine(Values("paris_icc_exif_xmp.png"),  // iCCP zTXt zTXt IDAT
```

Comment:
Optional: It would be good to add a README file to tests/data/ to describe the properties of the test files that we depend on. For example, we can document the metadata that paris_icc_exif_xmp.png has and the formats/chunks the metadata are stored in.

Reply:
It can be done in another PR.